test/controllers/todos_controller_test.rb

```diff
@@ -1,7 +1,69 @@
 require "test_helper"
 
 class TodosControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+  @created_todo
+  setup do
+    @valid_params={
+      title: "Valid Title",
+      status: "created",
+      is_completed: false
+    }
+    @invalid_params={
+      title: "title for invalid",
+      is_completed: false
+    }
+    @todo = Todo.create!(@valid_params)
+  end
+
+  test "#index should return response :ok" do
+    get todos_url
+    assert_response :ok
+  end
+
+  test "#index should return an Array" do
```

Comment:
Is the index returning the single Todo we would expect to see? I might check that as well.